src/views/waterUse/WaterUse.vue

```diff
@@ -78,23 +78,28 @@
         isAboutMapInfoBoxOpen: true,
         svg: "svg_map_te_spring",
         barchart: "svg_bars_te",
-        useParemeter: "te",
+        useParameter: "te",
         season: "spring"
       }
     },
+    computed: {
+      checkIfSVGMapIsRendered() {
+        console.log("Water Use Function");
```

Comment:
you sure you want that in there?

Reply:
Currently yes, as I am still testing to see if it actually works.